libs/libc/stdlib/lib_assert.c

```diff
@@ -0,0 +1,38 @@
+/****************************************************************************
+ * libs/libc/stdlib/lib_assert.c
```

Comment:
Is this the right place for this function?

Reply:
Do you have a better location? I select this location because lib_abort.c put here too. Or you like we create a new folder libs/libc/assert like other libc implementation.

Reply:
`abort()` is part of stdlib so it should be there.  I don't know about `_assert()`.  I honestly don't know about a better location.  Maybe as you suggested a new folder under libc?

Reply:
Done, move to libs/libc/assert